apps/climatemappedafrica/src/components/LineClampedRichTypography/LineClampedRichTypography.js

```diff
@@ -0,0 +1,54 @@
+import { RichTypography } from "@commons-ui/core";
```

Comment:
Forgot to ask, shouldn't this be `@commons-ui/next`? I believe `climatemappedafrica` (and `pesayetu`) are still using the old @commons-ui/core. Not sure if that one had `RichTypography` and if it did, not sure if it works like the current `RichTypography`.

```suggestion
import { RichTypography } from "@commons-ui/next";
```